future/dryrun.go

```diff
@@ -0,0 +1,114 @@
+package future
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/algorand/go-algorand-sdk/client/v2/algod"
+	"github.com/algorand/go-algorand-sdk/client/v2/common/models"
+	"github.com/algorand/go-algorand-sdk/crypto"
+	"github.com/algorand/go-algorand-sdk/types"
+)
+
+const (
+	defaultAppId uint64 = 1380011588
+)
+
+// CreateDryrun creates a DryrunRequest object from a client and slice of SignedTxn objects and a default configuration
+// Passed in as a pointer to a DryrunRequest object to use for extra parameters
+func CreateDryrun(client *algod.Client, txns []types.SignedTxn, dr *models.DryrunRequest) (drr models.DryrunRequest, err error) {
```

Comment:
Instead of using `context.Background()` in the body of this function, could you make the context an argument? That way the caller has control of it.